crates/re_space_view_spatial/src/contexts/transform_context.rs

```diff
@@ -64,11 +64,11 @@ impl Default for TransformContext {
 }
 
 impl ViewContextSystem for TransformContext {
-    fn archetypes(&self) -> Vec<ArchetypeDefinition> {
+    fn all_required_components(&self) -> Vec<IntSet<ComponentName>> {
         vec![
-            vec1::vec1![Transform3D::name()],
-            vec1::vec1![Pinhole::name()],
-            vec1::vec1![DisconnectedSpace::name()],
+            std::iter::once(Transform3D::name()).collect(),
+            std::iter::once(Pinhole::name()).collect(),
+            std::iter::once(DisconnectedSpace::name()).collect(),
```

Comment:
these are not all required - only the first one is

Reply:
These are different sets -- any one of them is sufficient to trigger the system